test/method/testpopulation.py

```diff
@@ -126,8 +126,54 @@ def testsumspins(self):
         totalspin = sum(self.analysis.fragspins)
         self.assertAlmostEqual(totalspin, formalspin, delta=1.0e-3)
 
-
-tests = [GaussianMPATest, GaussianLPATest, GaussianCSPATest]
+class GaussianBPATest(unittest.TestCase):
+    """Bickelhaupt Population Analysis test"""
+    
+    def setUp(self):

```

Comment:
Just to be prudent, add a `super(GaussianBPATest, self).setUp()` call. We should be doing that everywhere.